src/Console.gd

```diff
@@ -36,6 +36,9 @@ var isConsoleShown = true setget _setProtected
 # @var  bool
 var submitAutocomplete = true
 
+# @var bool
+var consumeInput = false
```

Comment:
I think it would be better making this true by default.

Reply:
i think you're right. Just wasn't sure if this was inteded behavior.
will change it true by default